site/docs/v1/tech/core-concepts/users.adoc

```diff
@@ -74,21 +74,9 @@ The database search engine enables fuzzy search against the following fields of
 
 image::user-search-database.png[User Search with Database Search Engine,width=1200,role=shadowed bottom-cropped]
 
-If you don't need advanced searching capabilities, you may be able to use the database search engine for large installations. This is not a use case FusionAuth tests, so ensure you provision your database with enough resources and benchmark your typical use cases.
-
-==== Limitations
-
-You may add a `*` character to wildcard match any character, including none. So `*piedpiper` will match `piedpiper` and `thepiedpiper`. You may put the wildcard at any location in a search string.
-
-All search terms are converted to lowercase and compared with lowercase values.
-
-Regular expressions, ranges and other complicated queries can not be used.
-
 === Elasticsearch Search Engine
 
-Leveraging Elasticsearch for the user search engine, enables advanced search capabilities on more numerous and granular data and a performance improvement for user search.
-
-The Elasticsearch search engine is appropriate for systems that are dependent on the link:/docs/v1/tech/apis/users#search-for-users[User Search APIs], are expected to have a large number of active users, and requires a more tactical search than is provided by the database search engine.
+Leveraging Elasticsearch for the user search engine enables advanced search capabilities on more numerous and granular data and a performance improvement for user search.
```

Comment:
" and granular data and a performance improvement" -> " and granular data. It also provides a performance improvement."